badges/badges.md

```diff
@@ -1,26 +1,25 @@
-# Badges!
+# Badges
 
-### What are badges?
-The badges are a complement system to the Ladder - while ladder outlines career paths, the badges reward behaviors and achievements we respect and appreciate in our fellows.
+### What are the badges?
+The badges are a complementing tool to the Ladder - while ladder outlines career paths, badges reward behaviors and achievements we respect and appreciate in our team members.
 
 ### How can I create a badge?
-Simple. Badges must meet the following criteria:
-#### 1. A badge achievement must be a measurable, objective, pass fail criteria.
-###### Example Badge: *Good presenter*
-Do: Presented to an audience of at least 50 people
-Don't: Had a nice, well structured presentation
+Simply. Badges must meet the following criteria:
+#### 1. Badge achievement must be a measurable, objective, and pass-fail criteria.
+###### Example: *Good speaker*
+Surely: Prepared an educative talk for an audience of ~50+ people
+Not really: Uploaded a few well-structured presentations
 
-#### 2. A badge must be achievable by one person in reasonable time
-###### Example Badge: *CR Knight*
-Do: Performed 1000 code reviews
-Don't: Invested 100000 hours reviewing code
+#### 2. A badge must be achievable by one person in a reasonable time
+###### Example: *Code Review Knight*
+Surely: Had objective insights that helped many people get on the right track
```

Comment:
Do's and don'ts are far more easily recognizable than Surely and Not really

Reply:
My problem with Do't and don'ts is that it is more open to interpretation. What does it mean in this instance? "DON'T do it" or "DO it but it doesn't recognise you as a badge bearer"

Reply:
When you write a badge:
Do - Write a clear objective measurement
Don't: Write something subjective and unclear

Reply:
fixed